src/OrleansTestingHost/TestingSiloHost.cs

```diff
@@ -661,13 +677,29 @@ public static SiloHandle StartOrleansSilo(TestingSiloHost host, Silo.SiloType ty
 
             // Load initial config settings, then apply some overrides below.
             ClusterConfiguration config = new ClusterConfiguration();
-            if (options.SiloConfigFile == null)
+            try
             {
-                config.StandardLoad();
+                if (options.SiloConfigFile == null)
+                {
+                    config.StandardLoad();
```

Comment:
Even if we remove the configuration files for 1.2.0, when they update from am older version nuget will not remove the files automatically,  so whatever they had,  it will still be there and continue to work.
Anyway, if you feel otherwise I'll make the change,  I'm just trying to not break behavior in silent ways that they can only notice by reading the change log.